wled00/wled_eeprom.cpp

```diff
@@ -102,7 +102,7 @@ void loadSettingsFromEEPROM()
   busses.add(bc);
 
   notifyButton = EEPROM.read(230);
-  notifyTwice = EEPROM.read(231);
+  if (EEPROM.read(231)) ubUdpNumRetries = 2;
```

Comment:
Different than default. Intentional and why?

Reply:
Same issue as this comment https://github.com/Aircoookie/WLED/pull/2830#discussion_r998020179. The intention here is to carry over the previous settings to this new system but it should set retries to one.